platforms/osx/src/main.mm

```diff
@@ -13,11 +13,9 @@
 
 std::string sceneFile = "scene.yaml";
 
-std::string markerStyling = "{ style: 'points', interactive: true, color: 'white', size: [25px, 25px], order: 100, collide: false }";
+std::string markerStyling = "touch.point.icons";
```

Comment:
That commit addresses part of my comment, but `draw` is still omitted from the path without explanation.